Source/Scene/Scene.js

```diff
@@ -1748,6 +1787,88 @@ define([
         return object;
     };
 
+    var scratchPickDepthPosition = new Cartesian3();
+    var scratchMinDistPos = new Cartesian3();
+    var scratchPackedDepth = new Cartesian4();
+    var packedDepthScale = new Cartesian4(1.0, 1.0 / 255.0, 1.0 / 65025.0, 1.0 / 160581375.0);
+
+    /**
+     * Returns the cartesian position reconstructed from the depth buffer and window position.
+     *
+     * @param {Cartesian2} windowPosition Window coordinates to perform picking on.
+     * @param {Cartesian3} [result] The object on which to restore the result.
+     * @returns {Cartesian3} The cartesian position.
+     *
+     * @exception {DeveloperError} Picking from the depth buffer is not supported. Check Context.depthTexture for support.
+     * @exception {DeveloperError} 2D is not supported. An orthographic projection matrix is not invertible.
+     */
+    Scene.prototype.pickDepth = function(windowPosition, result) {
```

Comment:
I should have thought of this sooner, but don't you think this should be `pickPosition` since `pickDepth` is an implementation detail?